src/stories/components/Covid/Covid.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import styles from "./Covid.module.css";
+
+type CovidProps = {
+  className?: string;
+};
+
+export default function Covid({ className }: CovidProps) {
```

Comment:
Maybe you could think about renaming the component so it is clear what the component actually is, e.g. `CovidLink`.

Reply:
Hi, thank u, that's also a good idea. Your suggestion is easier to understand, so i changed it. 💯